base_local_planner/src/jerk_cost_function.cpp

```diff
@@ -99,9 +99,9 @@ void JerkCostFunction::calculateAccelerations(Trajectory* traj, Eigen::Vector3f
     (*linear_accel) = (traj->xv_ - vel[0]) / traj->time_delta_;
     (*angular_accel) = (traj->thetav_ - vel[2]) / traj->time_delta_;
   }
-  else
+  else if (traj->time_delta_ < 0)
   {
-    ROS_WARN("Non-positive time delta in jerk cost function. %f.  %s", traj->time_delta_, msg.c_str());
+    ROS_WARN("Negative time delta in jerk cost function. %f.  %s", traj->time_delta_, msg.c_str());
```

Comment:
I've seen this a few times, is it something that should never happen or can occasionally happen?  If never we may want to make it an error, or if it occasionally happens possibly add some sort of error threshold if it may cause problems (count > x over n min?)